tests/aiq/llm_providers/test_langchain_agents.py

```diff
@@ -0,0 +1,98 @@
+# SPDX-FileCopyrightText: Copyright (c) 2025, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import pytest
+from langchain_core.messages import AIMessage
+from langchain_core.prompts import ChatPromptTemplate
+
+from aiq.builder.framework_enum import LLMFrameworkEnum
+from aiq.builder.workflow_builder import WorkflowBuilder
+from aiq.llm.aws_bedrock_llm import AWSBedrockModelConfig
+from aiq.llm.nim_llm import NIMModelConfig
+from aiq.llm.openai_llm import OpenAIModelConfig
+
+
+@pytest.mark.integration
+@pytest.mark.asyncio
```

Comment:
`@pytest.mark.asyncio` is unnecessary. Just labeling the function `async` is enough.